charts/manila-csi-plugin/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: v1.28.0
 description: Manila CSI Chart for OpenStack
 name: openstack-manila-csi
-version: 2.29.0-alpha.1
+version: 2.29.0-alpha.4
```

Comment:
Why do we jump 3 versions here?

Reply:
I updated the wrong file, I corrected the mistake on the latest commit